utils/set-unomi-root-password.yml

```
@@ -18,15 +18,52 @@ onInstall:
       var newUrl = "${baseUrl}".split('/')
       newUrl.pop();
       newUrl.pop();
-      return {result: 0, "universal_url": newUrl.join('/')};
+      return {result: 0, "jelastic_unomi_url_url": newUrl.join('/')};
```

_Not only_ because I dont like `pop()` :trollface:  :
```suggestion
      return {result: 0, "jelastic_unomi_url_url": "${baseUrl}".match(/(.*)(\/[A-Za-z0-9._-]+){2}$/)[1]};
```

or
```suggestion
      return {result: 0, "jelastic_unomi_url_url": "${baseUrl}".split('/').slice(0,-2).join('/')};
```